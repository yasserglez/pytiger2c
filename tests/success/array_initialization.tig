let
	type ints = array of int
	
	function zero(): int = 
	    (print("Print this only one time.\n"); 
         0)
	
	var num_ints := 10
	var ints := ints[num_ints] of zero()
in
	for i := 0 to (num_ints - 1) do 
	    (printi(ints[i]);
	     print("\n"))
end
