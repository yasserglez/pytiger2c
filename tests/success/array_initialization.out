Print this only one time.
0
0
0
0
0
0
0
0
0
0
